handbook/engineering/search/index.md

```diff
@@ -65,14 +73,16 @@ Our current focus is documented in the [tracking issue for the current milestone
           - **Plan:** Search onboarding tour
           - **Owner:** Farhan
           - **Status:** In progress
+          - **Estimated completion**: TODO
       - It is hard for users to quickly get to code they care about.
           - **Plan:** Enterprise homepage
           - **Owner:** Farhan, Juliana
           - **Status:** In progress
-    - **Outcomes:**
-      - New users introduced to Sourcegraph are able to quickly run searches that show them the value of Sourcegraph.
-      - Users can run searches over code they care about more quickly.
-      - Sourcegraph surfaces code and searches users care about.
+         - **Estimated completion**: TODO
```

Comment:
@attfarhan @limitedmage can you add an estimated completion milestone for this?

Reply:
```suggestion
         - **Estimated completion**: 3.21 release
```

I think we can get an initial release for this in time for 3.21, but this will be a part of the product that we will keep iterating on.

Reply:
I actually this can ship in this release (3.20), worst case behind a feature flag. @limitedmage what do you think is blocking us from shipping the enterprise homepage this release?

Reply:
I agree that we can likely do 3.20 with the feature flag off. Since 3.20 cut is tomorrow, there won't be time to polish any issues.

Reply:
> there won't be time to polish any issues

Then it sounds like 3.21 is the right milestone to have here :)